### DISCUSSION TITLE
Next.js 12 using Multi-zone Websocket connection issues

### DISCUSSION THREAD

**Discussion by stephenway** [vercel/next.js#30925] [Help]
I'm running a multi-zone setup and after upgrading to Next.js 12 on all zones except for the main home zone I'm getting the following warning:

`WebSocket connection to 'ws:<URL>/zoneName/_next/webpack-hmr' failed: WebSocket is closed before the connection is established.`

I'm not using any custom express or nginx setups in any of these zones. However I'm wondering if I should be to get this to work. Assuming the issue is that the traffic should be going to the port and not the home port + zone name?

**Comment by ebo-bpi** [comment]
same issue. mostly seen in Firefox. Chrome seems to not erroring about it.

**Comment by teovillanueva** [comment]
Are there any news on this?

**Comment by sioquim** [comment]
Is there an update to this one?

**Reply by teovillanueva** [reply]
I've made a PR for this #38455 a few days ago. Still no reviews 🙂
